.github/workflows/deploy-auth.yml

```
@@ -0,0 +1,59 @@
+name: Deploy Auth Service
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)